frontend/src/pages/WorkspaceLocked/customerStoryCard.styles.scss

```diff
@@ -0,0 +1,33 @@
+$component-name: 'customer-story-card';
+$ant-card-override: 'ant-card';
+$light-theme: 'lightMode';
+
+.#{$component-name} {
+	max-width: 385px;
+	margin: 0 auto; // Center the card within the column
+	margin-bottom: 24px;
+	border-radius: 6px;
+	transition: transform 0.3s ease, box-shadow 0.3s ease;
+	background-color: var(--bg-ink-400);
+	border: 1px solid var(--bg-ink-300);
+
+	.#{$light-theme} & {
```

Comment:
<img width="1915" alt="Screenshot 2024-08-23 at 11 21 49 AM" src="https://github.com/user-attachments/assets/1926f1c2-683a-4ed0-855d-346fb96ee9e5">


@sharpshooter90 : darkMode and lightMode class names are added to the root node. We can use it to nest individual classes.